labs/blockchain-explorer-iroha.md

```diff
@@ -0,0 +1,17 @@
+# Lab Name
+[Integration of Hyperledger Iroha into Hyperledger Explorer tool](https://wiki.hyperledger.org/display/INTERN/Integration+of+Hyperledger+Iroha+into+Hyperledger+Explorer+tool)
+
+# Short Description
+Internship project: [Integration of Hyperledger Iroha into Hyperledger Explorer tool](https://wiki.hyperledger.org/display/INTERN/Integration+of+Hyperledger+Iroha+into+Hyperledger+Explorer+tool).  
+Fork of [hyperledger/blockchain-explorer](https://github.com/hyperledger/blockchain-explorer) with support for Hyperledger Iroha.
+
+# Scope of Lab
+Code of [hyperledger/blockchain-explorer](https://github.com/hyperledger/blockchain-explorer) supported only Hyperledger Fabric and needed refactoring.  
+This fork supports only Hyperledger Iroha. It rebuilds backend part from scratch and refactors frontend part.
+
+# Initial Committers
+- https://github.com/turuslan
+- https://github.com/Pobepto
+
+# Sponsor
+Hyperledger Internship project
```

Comment:
Lab sponsor needs to be a member of the TSC, a maintainer of one of the projects, or a working group chair. Please provide a sponsor that meets these criteria and has agreed to sponsor the lab.